services/movie.py

```diff
@@ -1,13 +1,18 @@
 from django.db.models import QuerySet
 
+from django.db import transaction
+
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The same issue with the variable name `actors_ids`. Consider renaming it to `actor_ids`.